startup/10-motors.py

```diff
@@ -1,20 +1,20 @@
-from ophyd import Device, Component as Cpt, EpicsMotor
+from ophyd import MotorBundle, Component as Cpt, EpicsMotor
```

Comment:
I am not sure how useful `MotorBundle` is, as it's just a subclass of `Device`, with no extra additions, see its [current implementation](https://github.com/bluesky/ophyd/blob/a1d05d0df45d387e409915ad7c5911936c2cc6a8/ophyd/epics_motor.py#L359-L364) for details.

I don't know the history of `MotorBundle`, maybe I am missing something. @tacaswell, @danielballan, @klauer?

Reply:
I found it, by accident, mentioned in the ophyd documentation, it sounded useful. If it shouldn't be used I can revert that

Reply:
It may be vestigial at this point - I see it being used, but no special handling anywhere? 🤷‍♂️ Its author @tacaswell would probably know better